### PR TITLE
Added Wide Residual Network 

### PR DESCRIPTION
**Pull request by titu1994**
Added Wide Residual Network to applications and an example script to train the WRN-28-8 model.

### CODE REVIEW

File: keras_contrib/applications/wide_resnet.py [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+        width: Width of the network.
+        dropout: Adds dropout if value is greater than 0.0
+
+    Returns:a Keras Model
+
+    '''
+
+    N = (depth - 4) // 6
+
+    x = __initial_conv(img_input)
```
Comment (kemaswill) [inline keras_contrib/applications/wide_resnet.py]: Better to change '__initial_conv', '__conv1_block', '__conv2_block', __conv3_block to '__conv1_block', __conv2_block, '__conv3_block', '__conv4_block' respectively to keep the same as in the paper.

File: keras_contrib/applications/wide_resnet.py [hunk 2]
```diff
@@ -0,0 +1,300 @@
...
+        include_top: Flag to include the last dense layer
+        depth: Depth of the network. Compute N = (n - 4) / 6.
+               For a depth of 16, n = 16, N = (16 - 4) / 6 = 2
+               For a depth of 28, n = 28, N = (28 - 4) / 6 = 4
+               For a depth of 40, n = 40, N = (40 - 4) / 6 = 6
+        width: Width of the network.
+        dropout: Adds dropout if value is greater than 0.0
+
+    Returns:a Keras Model
+
```
Comment (kemaswill) [inline keras_contrib/applications/wide_resnet.py]: Delete empty line here.

File: examples/wide_resnet_cifar10.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
+from __future__ import absolute_import
```
Comment (kemaswill) [inline examples/wide_resnet_cifar10.py]: Add the performance as comments in the beginning of the script.

**Comment by kemaswill** [comment]
Please change the file name and add the performance in the beginning.

**Comment by titu1994** [comment]
I'm going to have to delete the repo and start over again cause of a ton of merge errors. For now, please merge. I will add another PR to fix the name.

Or I can close this for now and restart.

**Pull request closed without merging**